### PR TITLE
feat: add in spinner for all screens

### LINKED ISSUE #210: Loading ...  use Spinner.svg
Bildschirmfoto 2021-09-11 um 17 00 57

reminder : I did a spinner / loader  in ui3 branch . we could embed it when the "Loading..." appears .
https://raw.githubusercontent.com/dcgtc/dgrants/richard-ui3/app/src/assets/spinner.svg

embed centered above the "loading ..."  text  in like 64x64px for now

### PR DESCRIPTION
**Pull request by thelostone-mc**
#### Description

- add spinner asset provided by @scco 
- add new LoadingSpinner component 
- update all views to use LoadingSpinner where applicable

###### Note:
it has not been integrated to the listed components as they are being changed by and I didn't want to introduce conflicts
- GrantRoundList.vue ( by @carletex )
- Home.vue (@mk1r )
 
Untitled

closes #210

### CODE REVIEW

**Comment by scco** [comment]
thanks.  looks good to me after use regular font  width in grey-500  for "Loading..."

**Comment by carletex** [comment]
Looks great! I'll add it to `GrantRoundList.vue` on my PR once this is merged.

File: app/src/components/LoadingScreen.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      {{ text }}
+    </h2>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'LoadingScreen',
```
Comment (mds1) [inline app/src/components/LoadingScreen.vue:14]: Nit: Rename to `LoadingSpinner` everywhere, since this is a component and not a full page screen

Comment (thelostone-mc) [reply app/src/components/LoadingScreen.vue:14]: Oooo nice one

**Review by mds1 — APPROVED** [review]

**Review by gdixon — APPROVED** [review]
This looks awesome! 🚀

**Pull request merged**